### PR TITLE
Add additional logging

### PR DESCRIPTION
**Pull request by jawwadmukhtargmail**

### CODE REVIEW

File: CommunicationService/CommunicationService.AzureFunction/ProcessJobQueue.cs [hunk 1]
```diff
@@ -22,10 +22,22 @@ public ProcessJobQueue(IMessageFactory messageFactory)
...
             Dictionary<int, string> recipients = message.IdentifyRecipients(sendCommunicationRequest.RecipientUserID, sendCommunicationRequest.JobID, sendCommunicationRequest.GroupID);
 
+            if (recipients.Count == 0)
+            {
+                log.LogInformation("No recipients identified");
+            }
+            else
+            {
+                var rec = JsonConvert.SerializeObject(recipients);
+                log.LogInformation($"Recipients { rec}");
```
Comment (mike-f50) [inline CommunicationService/CommunicationService.AzureFunction/ProcessJobQueue.cs:38]: Is this PII?

File: CommunicationService/CommunicationService.AzureFunction/RequestCommunication.cs [hunk 2]
```diff
@@ -32,7 +33,8 @@ public RequestCommunication(IMediator mediator)
         {
             try
             {
-                log.LogInformation("C# HTTP trigger function processed a request.");
+                var request = JsonConvert.SerializeObject(req);
+                log.LogInformation($"RequestCommunicationRequest {request}");
```
Comment (mike-f50) [inline CommunicationService/CommunicationService.AzureFunction/RequestCommunication.cs:37]: Again, PII?

File: CommunicationService/CommunicationService.AzureFunction/ProcessMessageQueue.cs [hunk 3]
```diff
@@ -26,6 +26,7 @@ public ProcessMessageQueue(IMessageFactory messageFactory,ICosmosDbService cosmo
         [FunctionName("ProcessMessageQueue")]
         public void Run([ServiceBusTrigger("message", Connection = "ServiceBus")]string myQueueItem, ILogger log)
         {
+            log.LogInformation($"myQueueItem {myQueueItem}");
```
Comment (mike-f50) [inline CommunicationService/CommunicationService.AzureFunction/ProcessMessageQueue.cs:29]: PII?

File: CommunicationService/CommunicationService.AzureFunction/ProcessJobQueue.cs (hunk 1 above)
Comment (jawwadmukhtargmail) [reply CommunicationService/CommunicationService.AzureFunction/ProcessJobQueue.cs:38]: just id's

File: CommunicationService/CommunicationService.AzureFunction/RequestCommunication.cs (hunk 2 above)
Comment (jawwadmukhtargmail) [reply CommunicationService/CommunicationService.AzureFunction/RequestCommunication.cs:37]: again just id's

File: CommunicationService/CommunicationService.AzureFunction/ProcessMessageQueue.cs (hunk 3 above)
Comment (jawwadmukhtargmail) [reply CommunicationService/CommunicationService.AzureFunction/ProcessMessageQueue.cs:29]: just id's

**Pull request merged**